pkg/util/reconciler.go

```diff
@@ -85,29 +86,68 @@ func (r *ReconcilerBase) GetEgressServices(ctx context.Context, owner client.Obj
 	}
 
 	for _, outboundRule := range accessPolicy.Outbound.Rules {
-		if outboundRule.Namespace == "" {
-			outboundRule.Namespace = owner.GetNamespace()
-		}
+		namespaces := []string{}
+
+		if outboundRule.NamespacesByLabel == nil {
```

Comment:
On a general note - what happens when a user specifies both NamespacesByLabel and Namespace in a rule? Should we add some validation and give the user some feedback if that happens? And maybe add a test?

Reply:
Yeah, we should. Namespace has priority over NamespacesByLabel, as stated in the documentation. The code reflects this, but there is no feedback anywhere if both are supplied

Reply:
Fixed in https://github.com/kartverket/skiperator/commit/42830ccf3e463a8e01fd291f9f91d1092e4b3521